spec/index.html

```diff
@@ -9722,7 +9722,7 @@ <h5>Set Functions</h5>
           <section id="defn_aggCount">
             <h5>Count</h5>
             <p>Count is a SPARQL set function which counts the number of times a given expression
-              has a bound, and non-error value within the aggregate group.</p>
+              has a bound, and non-error, value within the aggregate group.</p>
```

Comment:
The placement of this new comma reads a bit confusing now.

Alternatively, we could choose to not add this comma, and also remove the comma after `bound`.

Reply:
How about simply "has a value within the aggregate group." 

It says a few lines down in the defn "remove error elements from N".